src/placeos-rest-api/controllers/metadata.cr

```diff
@@ -10,15 +10,24 @@ module PlaceOS::Api
     before_action :check_support, only: [:update, :update_alt, :destroy]
     before_action :find_zone, only: [:children]
 
+    # Allow guest access to system details of a single room
+    skip_action :check_oauth_scope, only: [:show, :children_metadata]
+
     getter zone : Model::Zone?
 
     # Fetch metadata for a model
     #
     # Filter for a specific metadata by name via `name` param
     def show
-      parent_id = params["id"]
+      zone_id = params["id"]
```

Comment:
This controller is meant to be generic to control systems and zones, hence the `parent_id`.
I believe `children` is specific to a `Zone` as there's no concept of hierarchy with `System`s

Reply:
oh right, forgot about that - will update this